minecraft_servers/distaria/manifest.json

```diff
@@ -10,8 +10,13 @@
       "twitter": "DistariaNetwork",
       "discord": "https://discord.gg/asYBeq8QYD",
       "teamspeak": "distaria.net",
+      "instagram": "distarianet",
       "web_support": "https://distaria.net/forum/index.php?board/4-support/"
     },
+    "brand": {
+      "text": "#306fdb",
+      "background": "#306fdb"
```

Comment:
I'm sure that won't look so good 😬
It's like putting white text on a white background. 😅
I guess you have to change the text color.

Reply:
oh i now i check what the background color is